app/src/main/res/values/strings.xml

```diff
@@ -1083,6 +1084,50 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
+    <string name="quest_smoothness_description_good_paving_stones">Shallow or narrow gaps</string>
+    <string name="quest_smoothness_description_good_sett">Unusually flat stones, regular pattern, very shallow or narrow gaps</string>
+
+    <string name="quest_smoothness_description_intermediate_paved">Unusually rough surface, wider gaps, cracks, shallow ruts, potholes or other damage</string>
+    <string name="quest_smoothness_description_intermediate_paving_stones">Wider gaps, possibly some displaced stones or other damage</string>
+    <string name="quest_smoothness_description_intermediate_sett">Flat stones or with square edges, shallow or narrow gaps</string>
+    <string name="quest_smoothness_description_intermediate_compacted_gravel">Well maintained, at most shallow bumps or potholes</string>
+
+    <string name="quest_smoothness_description_bad_paved">Big gaps, potholes, ruts, crumbled paving or other damage</string>
+    <string name="quest_smoothness_description_bad_paving_stones">Displaced stones, big gaps or other damage</string>
+    <string name="quest_smoothness_description_bad_sett">Weathered stone or irregular shapes, big gaps or damage</string>
+    <string name="quest_smoothness_description_bad_compacted_gravel">Some bigger gravel, bumps, ruts or potholes</string>
+
+    <string name="quest_smoothness_description_very_bad_paved">Deep potholes, ruts or other dangerous damage</string>
+    <string name="quest_smoothness_description_very_bad_sett">Big gaps, displaced, missing or irregularly shaped stones or dangerous damage</string>
+    <string name="quest_smoothness_description_very_bad_paving_stones">Missing and broken stones, gaps or other dangerous damage</string>
+    <string name="quest_smoothness_description_very_bad_compacted_gravel">Some large stones, deep potholes, bumps, ruts, erosion or other dangerous damage</string>
+
+    <string name="quest_smoothness_description_horrible">Properly usable only by off-road vehicles</string>
+    <string name="quest_smoothness_description_very_horrible">Properly usable only by specialized off-road vehicles</string>
+    <string name="quest_smoothness_description_impassable">Only passable on foot</string>
+
+    <string name="quest_smoothness_wrong_surface">Surface is different</string>
+    <string name="quest_smoothness_surface_value">The surface was tagged as</string>
+    <string name="quest_smoothness_obstacle">There is an obstacle</string>
+    <string name="quest_smoothness_obstacle_hint">Please provide a general answer for the smoothness. Single obstacles like kerbs, speed bumps or cattle grids can be ignored. Repeated or unexpected obstacles should count towards the answer.</string>
+    <string name="quest_smoothness_hint">🚲,🚗,🚙,… indicate for which vehicle the surface should be OK to use. What matters is the big picture rather than single damages.</string>
```

Comment:
```suggestion
    <string name="quest_smoothness_hint">🚲, 🚗, 🚙, … indicate for which vehicles the surface should be okay to use. What matters is the big picture rather than single damages.</string>
```